app/Jobs/SendEmptyWikibaseNotificationsJob.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Notifications\EmptyWikibaseNotification;
+use App\Wiki;
+use Carbon\CarbonImmutable;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+use Illuminate\Support\Facades\Log;
+
+class SendEmptyWikibaseNotificationsJob extends Job implements ShouldBeUnique
+{
+    public function handle (): void
+    {
+        $allWikis = Wiki::all();
+        foreach ($allWikis as $wiki) {
+            try {
+                $this->sendEmptyWikibaseNotification($wiki);
+            } catch (\Exception $exception) {
+                Log::error(
+                    'Failure polling wiki '.$wiki->getAttribute('domain').' for sitestats: '.$exception->getMessage()
+                );
+            }
+        }
+    }
+
+    public function sendEmptyWikibaseNotification (Wiki $wiki): void
+    {
+        //Calculate how many days has passed since the wikibase instance was first created
+        $createdAt = $wiki->created_at;
+        $now = CarbonImmutable::now();
+        $emptyWikibaseDays = $createdAt->diffInDays($now);
+
+        $firstEdited = $wiki->first_edited;
+
+        if ($firstEdited == null && $emptyWikibaseDays >= 30) {
+            $user = $wiki->wikiManagers()->first();
+            $user->notify(new EmptyWikibaseNotification($wiki->sitename));
+            $wiki->wikibaseNotificationSentRecord()->updateOrCreate(['notification_type'=>'empty_wikibase_notification']);
```

Comment:
I think it would make sense to collect the possible values for `notification_type` as static properties on the model class. Else this is very typo prone to just put them here and there and elswhere.